test/fixtures/standalone-config.yml

```diff
@@ -0,0 +1,90 @@
+---
+version: '2'
+services:
+  zookeeper:
+    image: confluentinc/cp-zookeeper:latest
+    labels:
+    - io.confluent.docker.testing=true
+    environment:
+      ZOOKEEPER_CLIENT_PORT: 2181
+      ZOOKEEPER_TICK_TIME: 2000
+
+  kafka:
+    image: confluentinc/cp-kafka:latest
+    labels:
+    - io.confluent.docker.testing=true
+    environment:
+      KAFKA_BROKER_ID: 1
```

Comment:
not really related to this pr, but do we know if there is a reason we aren't using automatic broker ids?